appveyor.yml

```diff
@@ -13,8 +13,8 @@ build: false
 
 test_script:
   - cd webrender_api
-  - cargo test --verbose
+  - cargo build --verbose
```

Comment:
similarly, no need to build everything separately, calling `cargo` from the crate root propagates to all workspace projects